packages/dd-trace/src/platform/node/metrics.js

```diff
@@ -267,6 +267,19 @@ function unpackHistogram (buffer) {
   }
 }
 
+const kGCTypeScavenge = 1 << 0
```

Comment:
This whole bit probably warrants a comment linking to the whatever v8 header these are defined in.